var/spack/repos/builtin/packages/py-onnxconverter-common/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOnnxconverterCommon(PythonPackage):
+    """ONNX Converter and Optimization Tools"""
+
+    homepage = "https://github.com/microsoft/onnxconverter-common"
+    pypi     = "onnxconverter-common/onnxconverter-common-1.9.0.tar.gz"
```

Comment:
The PyPI tarball doesn't seem to contain the `requirements.txt` file needed to build this package, I think you'll need to use a GitHub URL instead